README.md

```diff
@@ -247,10 +247,11 @@ The output format for Grype is configurable as well:
 grype <image> -o <format>
 ```
 
-Where the `format`s available are:
+Where the `formats` available are:
```

Comment:
nit: this should probably not be code-quoted at all

Reply:
Well, the `format` is mentioned as an arg, so it is worth to highlight it somehow. But the existing way it looks a bit weird for my taste, the `formats` looks better but make it less consistent IMO. 
So, should I get rid of quotes at all?

Reply:
I'd say to just get rid of the code quotes, as it's not exactly code or leave it as is -- it's just used in that example as _format_, not _formats_. Also, there's no `--format` option (it's `--output` / `-o`).